src/views/studio/studio-curator-invite.jsx

```diff
@@ -6,38 +6,71 @@ import classNames from 'classnames';
 import {FormattedMessage} from 'react-intl';
 
 import {acceptInvitation} from './lib/studio-member-actions';
+import {selectShowCuratorInvite} from '../../redux/studio-permissions';
 
-const StudioCuratorInvite = ({onSubmit}) => {
+const StudioCuratorInvite = ({showCuratorInvite, onSubmit}) => {
     const [submitting, setSubmitting] = useState(false);
-    const [error, setError] = useState(null);
+    const [accepted, setAccepted] = useState(false);
+    const [error, setError] = useState(false);
+
+    if (!showCuratorInvite) return null;
+
+    if (error) {
+        return (
+            <div className="studio-invitation studio-info-box studio-info-box-error">
+                <div className="studio-invitation-msg">
+                    <FormattedMessage id="studio.curatorInvitationError" />
+                </div>
+            </div>
+        );
+    }
+
+    if (accepted) {
+        return (
+            <div className="studio-invitation studio-info-box studio-info-box-success">
+                <div className="studio-invitation-msg">
+                    <FormattedMessage id="studio.curatorInvitationAccepted" />
+                </div>
+            </div>
+        );
+    }
 
     return (
-        <div>
+        <div className="studio-invitation studio-info-box">
+            <div className="studio-invitation-msg">
+                <FormattedMessage id="studio.curatorInvitation" />
+            </div>
             <button
-                className={classNames('button', {
+                className={classNames('studio-invitation-button button', {
                     'mod-mutating': submitting
                 })}
                 disabled={submitting}
                 onClick={() => {
```

Comment:
I thought we couldn't use arrow functions as JSX props...

Reply:
Technically, we can. `jsx-no-bind` eslint rule forbids that, but it's not like it breaks the code. (Bad for performance though)

And we eslint-disable'd the rule file-wide so it doesn't cause lint failure.